


try{
    console.log(notDefined)
} catch(err){
    console.log('notDefined doesnt exist')
}